scripts/generate_errors.pl

```diff
@@ -46,7 +46,14 @@
 
 $/ = $line_separator;
 
-open(GREP, "grep \"define MBEDTLS_ERR_\" $include_dir/* |") || die("Failure when calling grep: $!");
+my @files = <$include_dir/*>;
```

Comment:
While we're at it, this should really be `$include_dir/*.h`, to avoid catching editor backup files, patch rejections, etc.

Reply:
While we're at it, shouldn't we at least try to exclude lines inside comments? Why doesn't the script even check if `define` is preceded by `#`?